src/VisualStudio/Core/Def/PackageRegistration.pkgdef

```diff
@@ -25,6 +25,11 @@
 "Title"="Enable experimental C#/VB LSP completion experience"
 "PreviewPaneChannels"="IntPreview,int.main"
 
+// Corresponds to WellKnownExperimentNames.LspPullDiagnosticsFeatureFlag
```

Comment:
this is required by the feature flags service to allow us to change the option via `EnableFeatureFlag`
Note that this is overridden by whatever the targeted notification defines.